### PR TITLE
Arduino dev

### PR DESCRIPTION
**Pull request by AndyPro720**
Tested version for arduino at competition

### CODE REVIEW

**Comment by AndyPro720** [comment]
@rish292 I have given you access to approve the PR, make sure to check the code, test it and then approve. For now merge PR is the way to go.

**Comment by AndyPro720** [comment]
After merging make sure to create the branch with same name

File: Arduino/Arduino_Uno_R3/src/main.c [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#include "ArduinoUno.c"
+#define Arduino_h
+
+void setup () 
+{ 
+  pinMode(13, OUTPUT);
+}
+void loop ()
+{
+  GPIO_Write('A', 13, 1);
```
Comment (rish292) [inline Arduino/Arduino_Uno_R3/src/main.c:10]: You have passed a char instead of a char * @AndyPro720

**Review by rish292 — CHANGES_REQUESTED** [review]
You have passed a char instead of a char * @AndyPro720

File: Arduino/Arduino_Uno_R3/src/main.c (hunk 1 above)
Comment (AndyPro720) [reply Arduino/Arduino_Uno_R3/src/main.c:10]: @rish292 char * isn't acting as a pointer but rather like a [string literal](https://stackoverflow.com/a/718482/12632622) here. So '' is valid here.

*Although for future definitions we will avoid that usage all together.

Comment (rish292) [reply Arduino/Arduino_Uno_R3/src/main.c:10]: Screenshot 2023-04-30 at 3 38 45 PM

Comment (rish292) [reply Arduino/Arduino_Uno_R3/src/main.c:10]: Screenshot 2023-04-30 at 3 40 55 PM

Comment (rish292) [reply Arduino/Arduino_Uno_R3/src/main.c:10]: ?? @AndyPro720

**Review by rish292 — COMMENTED** [review]
@AndyPro720

File: Arduino/Arduino_Uno_R3/src/main.c (hunk 1 above)
Comment (AndyPro720) [reply Arduino/Arduino_Uno_R3/src/main.c:10]: @rish292 It runs fine on my machine. Looking up few examples online, it seems to work fine on gcc but similar troubles have been reported with arduino compilation libs. So perhaps that.

Anyways, merge it for now. We'll iron it out by next PR.

**Review by rish292 — APPROVED** [review]

**Pull request merged**